docs/docs/api_reference/example_requests.md

```
@@ -26,3 +26,15 @@ curl -X POST http://127.0.0.1:9000/api/v1/tasks/execute \
   -H "Content-Type: application/json" \
   -d '{"name": "Build project", "description": "Set up repo"}'
 ```
+
+## Get Session-Scoped Token Usage
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)